mistral/pixtral-12b/README.md

```diff
@@ -0,0 +1,81 @@
+# Mixtral 8x7B Instruct Truss
+
+This is a [Truss](https://truss.baseten.co/) for Pixtral 12B. Pixtral 12B parameter language model released by [Mistral AI](https://mistral.ai/) and is a multimodal (text + vision) LLM. This README will walk you through how to deploy this Truss on Baseten to get your own instance of it.
+
+
+## Deployment
+
+First, clone this repository:
+
+```sh
+git clone https://github.com/basetenlabs/truss-examples/
+cd mistral/pixtral-12b
+```
+
+Before deployment:
+
+1. Make sure you have a [Baseten account](https://app.baseten.co/signup) and [API key](https://app.baseten.co/settings/account/api_keys).
+2. Install the latest version of Truss: `pip install --upgrade truss`
+
+With `pixtral-12b` as your working directory, you can deploy the model with:
+
+```sh
+truss push --publish
+```
+
+Paste your Baseten API key if prompted.
+
+For more information, see [Truss documentation](https://truss.baseten.co).
+
+### Hardware notes
+
+You need one A100 to run Pixtral at `fp32`. If you need access to A100s, please [contact us](mailto:support@baseten.co).
```

Comment:
i think A100s are no longer gated, so you can remove this bit. Also worth double-checking if we're actually running at fp32 (normally this stuff is half-precision)